modules/caddyhttp/reverseproxy/healthchecks.go

```diff
@@ -51,9 +51,13 @@ type HealthChecks struct {
 // health checks (that is, health checks which occur in a
 // background goroutine independently).
 type ActiveHealthChecks struct {
-	// The URI path to use for health checks.
+	// The path to use for health checks.
+	// DEPRECATED: Use 'uri' instead.
 	Path string `json:"path,omitempty"`
```

Comment:
We should probably emit a warning if this is used, too (separate from Caddyfile). (It's OK if warning is emitted twice if Caddyfile is used too.)

Reply:
Right - I was thinking about that but wasn't sure if I should make it double up or not. Will do.